cmd/manager/main.go

```diff
@@ -71,12 +71,13 @@ func main() {
 	// Initialize cluster actuator.
 	clusterActuator := cluster.NewActuator(cluster.ActuatorParams{
 		Client:         cs.ClusterV1alpha1(),
-		LoggingContext: "[cluster actuator]",
+		LoggingContext: "[\033[32mcluster actuator\033[0m]",
 	})
 
 	// Initialize machine actuator.
 	machineActuator := machine.NewActuator(machine.ActuatorParams{
-		Client: cs.ClusterV1alpha1(),
+		Client:         cs.ClusterV1alpha1(),
+		LoggingContext: "[\033[31mmachine actuator\033[0m]",
```

Comment:
Same as above, colors should be configurable. We might also want to use a proper library that detects a terminal

Reply:
That's a feature for a different ticket. I'm removing the colors in a sec. Can you open an issue with this feature request?

Reply:
@vincepri a library for detecting the terminal won't really work here, since this is running in a pod, and the user's shell will be pulling the logs through the api server.

Reply:
To be clear I don't think we should add colors given that the support depends on user configuration and might not look pretty for everyone. Given that this is going away, then it's all good!